### ISSUE TITLE
old docs are still available and out of sync w/ published docs ("ghost pages")

### ISSUE THREAD

**Issue by foot** [weaveworks/weave-gitops#3556]
**Describe the bug**

E.g. https://docs.gitops.weave.works/docs/0.6.0/cluster-management/getting-started/ — reports the latest version is 0.10.2

From @LappleApple, June 2023: We have several user guide pages showing up on the website that are not listed in the repository. @luizbafilho and I checked yesterday and were stumped. Maybe they live in the cloud. Let's make sure these do not appear in the left-side menu--they're confusing, bring up an inaccurate left-side page menu, and won't look great in Google searches.

- [x]  Remove this [Introduction](https://docs.gitops.weave.works/docs/next/enterprise/intro/index.html) page. 
- [ ]  Remove the “Enterprise” section it fits under. 
- [x]  Remove this [Architecture](https://docs.gitops.weave.works/docs/next/architecture) page.
- [x]  Remove this [Upgrading](https://docs.gitops.weave.works/docs/next/enterprise/upgrading) page.
- [x]  Remove this [Troubleshooting](https://docs.gitops.weave.works/docs/next/troubleshooting/index.html) page. 
- [x]  Remove this [Examples](https://docs.gitops.weave.works/docs/next/examples/sock-shop/index.html) page.
- [ ] ~Check for latest update: https://github.com/facebook/docusaurus/issues/6434~

We should perhaps clean up old file sitting around on in the docs bucket.

**ACs:**
- We've marked versions as not-cacheable.
- Old docs versions are sidelined off from the latest doc version, stored in their own section.
- Old docs versions retain the same styling of the user docs.
- The component tagging at top right is gone.
- We have established a simple, clear process for storing the next-latest version of docs in the user docs area.
- Docs listed above no longer appear on the website.

**Comment by yiannistri** [comment]
https://docusaurus.io/docs/seo

**Comment by LappleApple** [comment]
I don't see how the current setup helps anyone -- it only causes confusion as we/they look for accurate, current, useable docs and end up going to old versions. After chatting with a few folks, here's a proposal:

- We make [this "Enterprise Releases" page](https://docs.gitops.weave.works/docs/next/enterprise/getting-started/releases-enterprise/) into a "Previous Releases" page and pull it out of the Enterprise folder, to belong to the main "Introducing Weave GitOps" section as a sibling page of the Enterprise and OSS page folders
- We add some language to that "Previous Releases" page that explicitly points people to the previous versions of user docs for past releases. They're available [via this route](https://github.com/weaveworks/weave-gitops/tree/main/website/versioned_docs)
- We get rid of this tagging at top right (attached image) 

<img width="183" alt="Screenshot 2023-06-09 at 18 46 04" src="https://github.com/weaveworks/weave-gitops/assets/2453640/f94d24d3-8f2b-4ece-9beb-f46dabbd5b4e">

Shared this proposal with @joe-terrible and got a +1. Will collect some more feedback through next few days, and let's decide by mid next week.

**Comment by LappleApple** [comment]
And @lizwarner-weave +1's

**Comment by enekofb** [comment]
+1

>We have established a simple, clear process for storing the next-latest version of docs in the user docs area.

great adding this one

**Comment by LappleApple** [comment]
Hey @Himangini, how's it going? Let me know if you need anything. cc @yiannistri

**Comment by Himangini** [comment]
> Hey @Himangini, how's it going? Let me know if you need anything. cc @yiannistri

@LappleApple I am working on it. :)

**Comment by Himangini** [comment]
@LappleApple @yiannistri I've opened a PR with the changes, PTAL https://github.com/weaveworks/weave-gitops/pull/3823

> We should perhaps clean up old files sitting around on in the docs bucket.
We need a separate ticket for this. While deleting the requested files in our prod bucket I noticed there are a lot of old files still lying around and it's a pain to do this manually. Ideally, we need to add this to our docs workflow so we cleanup the bucket objects when new versions are published. Also, see comment [here](https://github.com/weaveworks/corp/issues/3620#issuecomment-1621962017)
